downstream/modules/platform/proc-installing-the-aap-setup-bundle.adoc

```diff
@@ -19,8 +19,8 @@ The download and installation of the setup bundle needs to be located on the aut
 +
 ----
 $ tar xvf \
-   ansible-automation-platform-setup-bundle-2.2.1.tar.gz
-$ cd ansible-automation-platform-setup-bundle-2.2.1-1
+   ansible-automation-platform-setup-bundle-2.3.tar.gz
```

Comment:
Hi,

It seems that the latest version setup-bundle is `ansible-automation-platform-setup-bundle-2.3-1.1.tar.gz`. So you need to relate 2.3 with 2.3-1.1 if you want to use the current setup-bundle version of 2.3 release.

Or you can use an abstract word, such as `ansible-automation-platform-setup-bundle-<RELEASE_VERSION>.tar.gz`. I would recommend this, although the user would need to replace it accordingly.